gostream/ffmpeg/avutil/avutil.go

```diff
@@ -73,6 +73,40 @@ func ptr(buf []byte) *C.uint8_t {
 	return (*C.uint8_t)(unsafe.Pointer(h.Data))
 }
 
+// SetFrameFromImgMacroAlign sets the frame from the given image.YCbCr adding
+// line padding to the image to ensure that the data is aligned to the given boundary.
+func (f *Frame) SetFrameFromImgMacroAlign(img *image.YCbCr, boundary int) {
```

Comment:
Is there a reference or sample code we can link here so we remember how we came to this conclusion?

Reply:
Good call out. I have been looking at the driver code [here](https://github.com/raspberrypi/linux/blob/rpi-5.11.y/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c#L160).

What is confusing is that the driver says `bytesperline_align` is set to `32`. However, by empirically looking at the buffer size the driver is allocating you can see it requires `64` byte alignment. I also tested with 32 byte alignment and it does not work.

Reply:
I would just add that link and this comment in the code just in case it gives us issues later.